x-pack/legacy/plugins/infra/server/lib/adapters/framework/kibana_framework_adapter.ts

```diff
@@ -174,28 +175,33 @@ export class InfraKibanaBackendFrameworkAdapter implements InfraBackendFramework
   ) {
     const internalRequest = req[internalInfraFrameworkRequest];
     const server = internalRequest.server;
+    const getVisData = get(server, 'plugins.metrics.getVisData');
+    if (!isFunction(getVisData)) {
```

Comment:
How about sticking to the native type check? It doesn't seem to be much longer.

```suggestion
    if (typeof getVisData !== 'function') {
```